src/Nethermind/Nethermind.Evm/BlockExecutionContext.cs

```diff
@@ -31,6 +31,4 @@ public BlockExecutionContext(BlockHeader blockHeader, UInt256 forceBlobBaseFee)
         Header = blockHeader;
         BlobBaseFee = forceBlobBaseFee;
     }
-
-    public static implicit operator BlockExecutionContext(BlockHeader header) => new(header);
```

Comment:
Would it be possible to remove `BlockExecutionContext(BlockHeader blockHeader, UInt256 forceBlobBaseFee)` constructor or change it that is still passes the spec? So we could use the spec from `BlockExecutionContext` in `TransactionProcessor` and `EVM`? 

This can be done in separate PR.